data/Tusharjagi.json

```diff
@@ -17,6 +17,16 @@
       "name": "Follow me on GitHub",
       "url": "https://github.com/Tusharjagi",
       "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Medium",
+      "url": "https://medium.com/@tusharjagi",
+      "icon": "FaMedium"
```

Comment:
One suggestion please remove the `FaMedium` icon and add the `SiMedium` icon. You can check the changelog of the medium logo from [this blog](https://medium.com/blogging-guide/mediums-new-logo-2020-unfinished-ellipses-logomark-e24c059871e7).
``` suggestion
      "icon": "SiMedium"
```